ci-operator/step-registry/openshift/e2e/cert-rotation/test-rotation/openshift-e2e-cert-rotation-test-rotation-ref.yaml

```diff
@@ -0,0 +1,14 @@
+ref:
+  as: openshift-e2e-cert-rotation-test-rotation
```

Comment:
There will also be confusion regarding the naming of the steps here. We already have one `[openshift-e2e-cert-rotation-test](https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift/e2e/cert-rotation/test/openshift-e2e-cert-rotation-test-ref.yaml#L2)`

Reply:
Yes